album/LACM-4812.toml

```
@@ -0,0 +1,30 @@
+[album]
+album_id = "6730f4be-0954-47d1-936f-da98620a8208"
+title = "Love marginal"
+artist = "Printemps（高坂穂乃果（新田恵海）、南ことり（内田彩）、小泉花陽（久保ユリカ））"
+date = 2011-03-25
```

```suggestion
date = 2011-05-25
```

https://www.lantis.jp/release-item/LACM-4812.html